inferno-types/src/Inferno/Types/VersionControl.hs

```diff
@@ -94,6 +103,16 @@ instance Hashable VCObjectHash where
   hashWithSalt salt (VCObjectHash digest) =
     hashWithSalt salt $ convert @(Digest SHA256) @ByteString digest
 
+instance Serialize VCObjectHash where
+  get =
+    (getByteString 44)
```

Comment:
SHA256 hashes have 256 bits of output, which translates to 64 characters in hex. That's double the length of the 32-character MD5 hash in hex. Base64 encoded SHA256 characters are only 44 bytes in length (and the last = for padding is omitted anyway).